src/main/ruby/core/rubinius.rb

```diff
@@ -44,24 +44,29 @@ module Rubinius
 
   HOST_OS = Truffle::System.host_os
 
+  IS_BSD = HOST_OS == 'bsd'
+  IS_LINUX = HOST_OS == 'linux'
+  IS_SOLARIS = HOST_OS == 'solaris'
```

Comment:
These checks aren't the same as the ones they're replacing. Did you verify they work the same on various BSDs, Solaris, and Linux distributions? If not, I'd switch back to the regexp match.

Reply:
This is all already normalized by `Truffle::System.host_os`. If not, it should be fixed there.

Reply:
These work for the Solaris, Darwin and Linux we have in CI at the very least.

Reply:
Maybe we can check that the value of `host_os` is within one of the expected values we know about, in `Truffle::System.host_os`.

Reply:
I see. Looking at `BasicPlatform` it looks like the `IS_BSD` value would be wrong, but we don't officially support any BSD either.

Reply:
I fixed this by checking the 3 possibles platforms that match BSD, and removing the "unknown" platform entirely, have a look at 69e441c.